package.json

```diff
@@ -61,14 +61,16 @@
     "symbol-observable": "^1.2.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
+    "@babel/core": "7.0.0-beta.44",
```

Comment:
`@bael/core` and `babel-core` dependencies? Is it necessary to have both dependencies?

Reply:
This helps other tools to use peerDep with babel 6 or 7, in this case such tool is `jest`. You can read more about the issue [here](https://github.com/babel/babel-bridge)

Reply:
Thank you for this information.